### PR TITLE
Feat/browser

### PR DESCRIPTION
**Pull request by giovannic**
This adds:

* Support for the squire model's new seeding strategy
* Moves model running code into the squire repo through `run_deterministic_SEIR_model`
* Windows support
* A more comprehensive e2e test. We now have several scenarios that run on chrome. The threshold is set to the tightest that passes all scenarios. It can be re-evaluated if/when our modelling code is updated.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN R -e 'install.packages( \
   c("V8", "odin", "deSolve", "jsonlite", "remotes"))'
 
 RUN R -e 'library(remotes); install_github(c("mrc-ide/odin.js", \
-"mrc-ide/squire@squire_js"))'
+"mrc-ide/squire@f4c509b"))'
```
Comment (richfitz) [inline Dockerfile:9]: This looks like it's now just behind master? Do you want to point at a symbolic reference instead?

File: R/run.R [hunk 2]
```diff
@@ -1,4 +1,3 @@
-source('R/probs.R')
```
Comment (richfitz) [inline R/run.R:1]: ah, much nicer :)

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  # env vars that may need to be set, at least temporarily, from time to time
+  # see https://github.com/krlmlr/r-appveyor#readme for details
+  # USE_RTOOLS: true
+  # R_REMOTES_STANDALONE: true
+
+# Adapt as necessary starting from here
+
+build_script:
+  - R -e "install.packages(c('V8', 'odin', 'deSolve', 'jsonlite', 'remotes'), repos = 'https://cloud.r-project.org/')"
+  - R -e "remotes::install_github(c('mrc-ide/odin.js', 'mrc-ide/squire@squire_js'))"
```
Comment (richfitz) [inline appveyor.yml:26]: this is a different reference to the one above

File: appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Download script file from GitHub
```
Comment (richfitz) [inline appveyor.yml:1]: Out of curiosity, why appveyor? I thought that gh actions were the goto for new work now, and you already have them wired up here

File: Dockerfile (hunk 1 above)
Comment (giovannic) [reply Dockerfile]: Ah! This is mainline now, no need for the silly refs. Thanks

File: appveyor.yml (hunk 4 above)
Comment (giovannic) [reply appveyor.yml:1]: So [this](https://github.com/mrc-ide/squire_js/blob/feat/windows/.github/workflows/e2e.yml) was my attempt at getting it to run on gh actions.

After tinkering with Chrome/Chromedriver [versions](https://github.com/mrc-ide/squire_js/runs/608173190?check_suite_focus=true#step:7:70) for a bit too long, I thought it was more important to simply get our COVID model tested 😆 

The great thing about appveyor is that the workers have all of the browsers and correct drivers installed. When there's a `chrome/install` action, gh actions could compete.

Comment (richfitz) [reply appveyor.yml:1]: seems like a good shout :)

**Pull request merged**